README.md

```diff
@@ -0,0 +1,13 @@
+# REPLAPI Module
+
+Hi! And welcome to the REPLAPI(pronouned "rep-"ll"-"a"-"p"-"i") module!
+
+This module acceses your(and anyone elses replit stats)!
+
+This gives any users `BIO`, `NICKNAME`, and `FAVORITE LANGS`. 
```

Comment:
Uh, im planning on adding more, but ok

Reply:
ok, this is for GitHub, like a readme they opens when you open the repo

Reply:
and yeh, you can edit the readme later